kor-styles.css

```diff
@@ -18,7 +18,7 @@ html, :host, *[theme="light"] {
   --text-3: rgba(0, 0, 0, .20);
   --shadow-1: 0 2px 8px rgba(0,0,0,.1), 0 1px 4px rgba(0,0,0,.05);
 }
-*[theme="dark"] {
+html, :host, *[theme="dark"] {
```

Comment:
Hi Alex and thanks for the contribution. For this line, wouldn't `html, :host` conflict with the light theme definition and therefore overwrite it? The rules are identical so that would be my assumption.

Reply:
True, my bad. I'll take another look.

Reply:
Sorry, but I cannot see how this can be solved without having separate theme stylesheets...

Reply:
> Sorry, but I cannot see how this can be solved without having separate theme stylesheets...

I think so too :( then if you don't mind I will close the pull request for now.